docs/conops.md

```diff
@@ -1,3 +1,178 @@
 # Concept of Operations - `svc-cargo`
 
-:construction:
+<center>
+
+<img src="https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png" style="height:250px" />
+
+</center>
+
+Item | Value
+--- | ---
+Maintainer(s) | @arrow-air/services
+Stuckee | A.M. Smith ([@ServiceDog](https://github.com/ServiceDog))
+
+:exclamation: This document is intended for internal use.
+
+## 1.0 Project Description
+
+#### 1.0.1 Background
+
+This service provides an interface for customers to schedule and manage cargo shipments. 
```

Comment:
What do you mean by "manage"?  Are you expecting additional features that allow users to manage existing shipments like tracking or cancel or reroute?

Reply:
Modify (departure or destination ports, times), cancel, etc.

We already support cancellation

Reply:
Changed "manage" to "modify"

Reply:
Probably change it to "This service provides an interface for shippers to book, schedule and modify open shipments."?

These are flight plans have not picked up by operators.

Reply:
Updated to "This service provides an interface for shippers to book, modify, and cancel cargo shipments."